tests/test_date.nit

```diff
@@ -0,0 +1,13 @@
+import date
+
+var test  = new Date(2014, 01, 09)
+var test2 = new Date(2016, 01, 09)
+var test3 = new Date(2015, 01, 12)
+var res   = test.date_diff(test2,2)
```

Comment:
This one can be moved in the `date_diff` comment.

Example:
~~~
This can be moved in the Date class comment as a `docunit`. See http://nitlanguage.org/tools/nitunit/.

Example:
~~~
# The main class Date (or a better synopsis)
#
#     var d  = new Date(2014, 01, 09)
#     assert d.to_s == "2014/01/09"
class Date
~~~
# Compare two Date
# forms : 1 for day, 2 for mounth, 3 for years
#
#     var d1  = new Date(2014, 01, 09)
#     var d2 = new Date(2016, 01, 09)
#     assert d1.date_diff(d2, 2) == [expected result]
fun date_diff(d2: Date, forms: Int): Int
...
~~~